src/plusplus.coffee

```diff
@@ -65,10 +65,10 @@ module.exports = (robot) ->
     # if we got a score, then display all the things and fire off events!
     if score?
       message = if reason?
-                  if reasonScore > 1
-                    "#{name} has #{score} points, #{reasonScore} of which are for #{reason}."
-                  else
+                  if reasonScore == 1 or reasonScore == -1
```

Comment:
Oh, while we're at it, could this change to`if 1, name has {score} point, for {reason}`? (singular `point`)